src/Prolens.hs

```diff
@@ -601,19 +648,28 @@ prism' ctor match = prism ctor (\s -> maybe (Left s) Right $ match s)
 {-# INLINE prism' #-}
 
 
-{- | Traversal
+{- | 'Traversal' is 'Optic' which is composable data structure traversal.
```

Comment:
It is not quite clear what is composable traversal. Maybe rewrite it in verbs instead of the noun? 🙂